common/src/main/java/com/gregtechceu/gtceu/api/data/worldgen/bedrockfluid/FluidVeinWorldEntry.java

```diff
@@ -42,7 +43,7 @@ public void setOperationsRemaining(int amount) {
     }
 
     public void decreaseOperations(int amount) {
-        operationsRemaining = Math.max(0, operationsRemaining - amount);
+        operationsRemaining = ConfigHolder.INSTANCE.worldgen.infiniteBedrockOresFluids ? Math.max(0, operationsRemaining - amount) : operationsRemaining;
```

Comment:
I think this should probably be inverted:

```suggestion
        operationsRemaining = ConfigHolder.INSTANCE.worldgen.infiniteBedrockOresFluids ? operationsRemaining : Math.max(0, operationsRemaining - amount);
```